dev/tasks/r/github.packages.yml

```diff
@@ -82,14 +82,24 @@ jobs:
       {{ macros.github_checkout_arrow()|indent }}
       {{ macros.github_change_r_pkg_version(is_fork, '${{ needs.source.outputs.pkg_version }}')|indent }}
       {{ macros.github_install_archery()|indent }}
+
       - name: Build libarrow
         shell: bash
         env:
-          UBUNTU: {{ '${{ matrix.config.version}}' }}
+          SCCACHE_BUCKET: {{ '${{ secrets.SCCACHE_BUCKET }}' }}
+          SCCACHE_ID: {{ '${{ secrets.AWS_ACCESS_KEY_ID }}' }}
+          SCCACHE_KEY: {{' ${{ secrets.AWS_SECRET_ACCESS_KEY }}' }}

```

Comment:
wouldn't be better to call this `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`. I am not sure the renaming adds anything and the environment variable names used on archery are already the original ones.

Reply:
The idea was to make the archery command a bit more concise as archery emulates the docker `-e` param but does not support passing only the envvar name.

Reply:
Also, I like that this is explicit that these are for sccache, not for unrelated purposes. So I would vote to keep the `SCCACHE` prefix.